test/js-native-api/test_typedarray/test_typedarray.c

```diff
@@ -97,6 +97,14 @@ static napi_value External(napi_env env, napi_callback_info info) {
   return output_array;
 }
 
+
+static napi_value EmptyArrayBuffer(napi_env env, napi_callback_info info) {
+  void* null_data = NULL;
+  napi_value array_buffer;
+  NAPI_CALL(env, napi_create_arraybuffer(env, 0, &null_data, &array_buffer));
```

Comment:
Would 0-lengthed arraybuffer be necessarily a detached arraybuffer? Would an explicit detach operation make sense?
IIUC the third parameter `data` is an out value.

Reply:
Oh, good catch. I'm not sure so I guess I'd better change this to explicitly detach.
(According to the spec `null` is not guaranteed in this case https://tc39.es/ecma262/#sec-allocatearraybuffer)

Reply:
Done, I went with an explicit external NULL array buffer, PTAL.